homeassistant/components/image/services.yaml

```diff
@@ -0,0 +1,12 @@
+# Describes the format for available image services
+
+snapshot:
+  target:
+    entity:
+      domain: image
+  fields:
+    filename:
+      required: true
+      example: "/tmp/snapshot_{{ entity_id.name }}.jpg"
```

Comment:
The examples need to be updated for the new variables

Reply:
because it was corrected to have the entity id instead of the entity or something else?